### ISSUE TITLE
Error: Call to a member function format() on null

### ISSUE THREAD

**Issue by jmsche** [Guikingone/SchedulerBundle#121]
I guess it happens when the task never run before?

```
16:18:02 CRITICAL  [console] Error thrown while running command "scheduler:consume -vv". Message: "Call to a member function format() on null" ["exception" => Error { …},"command" => "scheduler:consume -vv","message" => "Call to a member function format() on null"]

In Scheduler.php line 151:
                                              
  [Error]                                     
  Call to a member function format() on null  
                                              

Exception trace:
  at /var/www/subs.leapt.io/vendor/guikingone/scheduler-bundle/src/Scheduler.php:151
 SchedulerBundle\Scheduler->SchedulerBundle\{closure}() at n/a:n/a
 array_filter() at /var/www/subs.leapt.io/vendor/guikingone/scheduler-bundle/src/Task/TaskList.php:92
 SchedulerBundle\Task\TaskList->filter() at /var/www/subs.leapt.io/vendor/guikingone/scheduler-bundle/src/Scheduler.php:151
 SchedulerBundle\Scheduler->getDueTasks() at /var/www/subs.leapt.io/vendor/guikingone/scheduler-bundle/src/Command/ConsumeTasksCommand.php:101
 SchedulerBundle\Command\ConsumeTasksCommand->execute() at /var/www/subs.leapt.io/vendor/symfony/console/Command/Command.php:288
 Symfony\Component\Console\Command\Command->run() at /var/www/subs.leapt.io/vendor/symfony/console/Application.php:992
 Symfony\Component\Console\Application->doRunCommand() at /var/www/subs.leapt.io/vendor/symfony/framework-bundle/Console/Application.php:96
 Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() at /var/www/subs.leapt.io/vendor/symfony/console/Application.php:291
 Symfony\Component\Console\Application->doRun() at /var/www/subs.leapt.io/vendor/symfony/framework-bundle/Console/Application.php:82
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /var/www/subs.leapt.io/vendor/symfony/console/Application.php:167
 Symfony\Component\Console\Application->run() at /var/www/subs.leapt.io/vendor/symfony/runtime/Runner/Symfony/ConsoleApplicationRunner.php:56
 Symfony\Component\Runtime\Runner\Symfony\ConsoleApplicationRunner->run() at /var/www/subs.leapt.io/vendor/autoload_runtime.php:35
 require_once() at /var/www/subs.leapt.io/bin/console:11
```

**Comment by xaviermarchegay** [comment]
fixed by https://github.com/Guikingone/SchedulerBundle/issues/116

Unfortunately, the latest tag is wrong so the fix is not released yet.

**Comment by Guikingone** [comment]
Yes, I'm waiting for https://github.com/Guikingone/SchedulerBundle/pull/120 to be done / merged to release `0.4.9` 🙂

**Comment by Guikingone** [comment]
`0.4.9` released, sorry for the delay 🙂
